### PR TITLE
Addition of Johnson SU pdf

### PR DESCRIPTION
**Pull request by marinang**

### CODE REVIEW

File: probfit/pdf.pyx [hunk 1]
```diff
@@ -114,6 +114,94 @@ cdef class HistogramPdf:
             return 0.0
 
 
+cdef class _JohnsonSU:
+    """
+    Normalized JohnsonSU.
```
Comment (eduardo-rodrigues) [inline probfit/pdf.pyx]: Tiny suggestion: add in the docstring a reference for the JohnsonSU PDF. So something like:

```
Normalized JohnsonSU [1]_.
…

References
------------

.. [1] https://en.wikipedia.org/wiki/Johnson%27s_SU-distribution
```

Comment (HDembinski) [reply probfit/pdf.pyx]: +1

File: probfit/pdf.pyx [hunk 2]
```diff
@@ -135,7 +135,13 @@ cdef class _JohnsonSU:
 
     .. math::
         \\xi = \\mu + \\lambda \\exp\\left(\\frac{\\tau^{2}}{2}\\right)\\sinh
-        \\left( \\nu \\tau \\right)
+        \\left( \\nu \\tau \\right)$
```
Comment (eduardo-rodrigues) [inline probfit/pdf.pyx]: Seems you added a spurious "$" by mistake. Otherwise all good, thanks.

**Comment by eduardo-rodrigues** [comment]
Thanks. Both @HDembinski and I had approved. I'm merging,

**Pull request merged**